dist/server.js

```diff
@@ -1,29 +1,29 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-var express = require('express');
-var app = express();
+var express = require("express");
 var api = require('./api');
 var project = require('../package.json');
+var app = express();
 app.use(express.json());
 app.use(api);
-app.get('/', function (req, res) {
+app.get('/', function (_req, res) {
```

Comment:
good Elixir convention :)

Reply:
Oh, it is? I think it was the TS language server yelling at me 😅

Reply:
it could be that too, i'm not very familiar with TS